doc/website/concepts/qos-policies.md

```diff
@@ -0,0 +1,36 @@
+# QoS policies
+
+Unlike other middlewares like for example [DDS used in ROS 2](https://docs.ros.org/en/galactic/Concepts/About-Quality-of-Service-Settings.html)
```

Comment:
This reads a bit weird. Depending on the emphasis one could read that iceoryx is like DDS. How about
```md
Unlike other middlewares, e.g. DDS used in ROS 2, iceoryx ...
```
or
```md
Compared to other middlewares like e.g. DDS used in ROS 2, iceoryx ...
```

Or maybe just remove the DDS part.